spec/controllers/orders_controller_spec.rb

```diff
@@ -0,0 +1,108 @@
+require 'rails_helper'
+require 'support/app_controller'
+
+RSpec.describe OrdersController, type: :controller do
+  let(:order_hash) do
+  {
+    status: "pending",
+    cc_name: "John Carlisle",
+    email_address: "jcarl@gmail.com",
+    mailing_address: "653 Gorge Way",
+    cc_number: 5110538084994719,
+    cc_exp: "06/18",
+    cc_cvv: "674",
+    zip: 19583
+  }
+  end
+
+  let(:order) do
+    Order.create!(order_hash)
+  end
+
+  let(:cart_status) do
+    @cart_status
+  end
+
+  describe "GET 'show'" do
+    it "renders the show view" do
+      get :show, id: order.id
+      expect(subject).to render_template :show
+    end
+  end
+
+  describe "GET 'cart'" do
+    it "sets cart status to empty if there is no order_id in session" do
+      get :cart, {}, {:order_id => nil}
+      expect(@cart_status).to eq(nil)
+      expect(@order).to eq(nil)
```

Comment:
Same as with the comment above, I don't think this `@order` is aware of the instance variable `@order` in orders_controller. It will always return nil. I'm actually not entirely sure if you can directly test/check instance variables from RSpec.